body.tex

```diff
@@ -85,37 +85,41 @@ \subsection{The Tools Used}
 \end{figure}
 
 \paragraph{MagicDraw}
-is the Rubin Observatory Modeling tool. It is used for maintaining the project model and for requirement management. 
-The Verification Elements are created in MagicDraw using the requirements as stating point, 
-then they are synchronized to and from Jira using \textbf{Syndeia}. 
-This ensures that the proper traceability between requirements and verification elements is in place, 
+is the Rubin Observatory Modeling tool. It is used for maintaining the project model and for requirement management.
+%not sure project model is descriptive enough
```

Comment:
I have asked Austin to provide a better phrasing here... and to review also the Syndeia paragraph.